package.json

```diff
@@ -36,9 +36,11 @@
     "url": "git://github.com/alexdavid/exprestive.git"
   },
   "scripts": {
+    "compress": "gulp",
```

Comment:
I call this "build" in a couple of other node projects. Unless you are actually compressing with uglify or something, these seems misnamed to me